draft-liu-teas-transport-network-slice-yang.txt

```diff
@@ -82,93 +82,160 @@ Copyright Notice
    topologies, where the network slices are defined in
-   [I-D.ietf-teas-ietf-network-slices].
+   [I-D.ietf-teas-ietf-network-slices].  The YANG model augments the
+   data model defined in [RFC8345] to enable a customer to express
+   desired service-level objectives (SLOs) and service-level
+   expectations (SLEs) over various constructs within the customized
+   topology.
 
    The defined data model is an interface between customers and
    providers for configurations and state retrievals, so as to support
    network slicing as a service.  Through this model, a customer can
-   learn the slicing capabilities and the available resources of the
-   provider.  A customer can request or negotiate with a network slicing
-   provider to create an instance.  The customer can incrementally
-   update its requirements on individual topology elements in the slice
-   instance, e.g., adding or removing a node or link, updating desired
-   bandwidth of a link, and retrieve the operational states of these
-   elements.  With the help of other mechanisms and data models defined
-   in IETF, the telemetry information can be published to the customer.
-
-   The YANG model defines technology-agnostic constructs common to
-   network slicing at network layers of different technologies, e.g.
-   IP/MPLS(-TP), OTN and WDM.  Therefore, this model may be used as a
-   common base model on which other network slicing models, such as
-   [I-D.ietf-ccamp-yang-otn-slicing], may augments with technology-
-   specific constructs.
+   request or negotiate with a network slicing provider to create an
+   instance.
+   The customer can incrementally update its requirements on individual
+   topology elements in the slice instance, e.g., adding or removing a
+   node or link, updating desired bandwidth of a link, and retrieve the
+   operational states of these elements.  With the help of other
+   mechanisms and data models defined in IETF, the telemetry information
+   can be published to the customer, too.
+
+
+
+
+
+
+
+
+
+Liu, et al.             Expires 31 December 2023                [Page 3]
+
+Internet-Draft      Network Slice Topology Data Model          June 2023
+
+
+   The YANG model defines constructs that are technology-agnostic to
+   network slicing built on network layers with different technologies
+   such as IP/MPLS, MPLS-TP, OTN and WDM optical.  Therefore, this model
+   can serve as a common and base model on which technology-specific
+   models for network slicing, such as
+   [I-D.ietf-ccamp-yang-otn-slicing], may be built.
 
    As described in Section 3 of
    [I-D.contreras-teas-slice-controller-models], the data model defined
-   in this document complements the data model defined in
-   [I-D.ietf-teas-ietf-network-slice-nbi-yang].  In addition to the
-   provider's view, the data model defined in this document models the
-   Type 2 service defined in [RFC8453].
+   in this document is optional for network slicing and complements the
```

Comment:
Not sure to have understood the meaning of this sentence"the data model defined in this document is optional for network slicing"